server/services/reportedAdjudicationsService.ts

```diff
@@ -855,4 +855,71 @@ export default class ReportedAdjudicationsService {
   async getAgencyReportCounts(user: User): Promise<AgencyReportCounts> {
     return new ManageAdjudicationsClient(user).getAgencyReportCounts()
   }
+
+  async getPrisonerLatestADMMovement(
+    prisonerNo: string,
+    overrideAgencyId: string,
+    user: User
+  ): Promise<OffenderBannerInfo> {
+    const [movementInfo, prisoner] = await Promise.all([
+      new PrisonApiClient(user.token).getMovementByOffender(prisonerNo),
+      new PrisonApiClient(user.token).getPrisonerDetails(prisonerNo),
+    ])
+    const moveToOverrideAgencyIdList = movementInfo.filter(prisonerMove => prisonerMove.toAgency === overrideAgencyId)
```

Comment:
need to grab first match.  its ordered by latest anyway.  maybe safer to use Array.prototype.find() ?